sw/host/provisioning/ft/src/main.rs

```diff
@@ -115,7 +115,8 @@ fn main() -> Result<()> {
     let transport = backend::create(&opts.init.backend_opts)?;
     transport.apply_default_configuration(None)?;
     let spi = transport.spi(&opts.console_spi)?;
-    let spi_console_device = SpiConsoleDevice::new(&*spi)?;
+    let device_console_tx_ready_pin = &transport.gpio_pin("IOA5")?;
```

Comment:
Do we want to make this configurable with a command-line argument?  We can default it to IOA5 in the CLI.

Reply:
oops, didn't mean to include that just yet. I will submit a follow up PR that updates the perso flow to use the indicator GPIO pin. Until then, I just removed this for now. This PR just adds a test that uses this new feature.